test/e2e/specs/specs-playwright/wp-calypso__coblocks-spec.js

```diff
@@ -0,0 +1,168 @@
+/**
+ * External dependencies
+ */
+import {
+	DataHelper,
+	LoginFlow,
+	NewPostFlow,
+	GutenbergEditorPage,
+	PublishedPostPage,
+	ClickToTweetBlock,
+	PricingTableBlock,
+} from '@automattic/calypso-e2e';
+
+describe( DataHelper.createSuiteTitle( 'Gutenberg: CoBlocks', { parallel: false } ), function () {
+	describe( DataHelper.createSubsuiteTitle( 'Click to Tweet', { parallel: true } ), function () {
+		let gutenbergEditorPage;
+		let clickToTweetBlock;
+		const blockName = 'Click to Tweet';
+		const tweet =
+			'The foolish man seeks happiness in the distance. The wise grows it under his feet. — James Oppenheim';
+
+		it( 'Log in', async function () {
+			const loginFlow = new LoginFlow( this.page, 'gutenbergSimpleSiteUser' );
+			await loginFlow.logIn();
+		} );
+
+		it( 'Start new post', async function () {
+			const newPostFlow = new NewPostFlow( this.page );
+			await newPostFlow.newPostFromNavbar();
+			gutenbergEditorPage = await GutenbergEditorPage.Expect( this.page );
+		} );
+
+		it( 'Enter post title', async function () {
+			await gutenbergEditorPage.enterTitle( blockName );
+		} );
```

Comment:
Is this step necessary?

Reply:
Since we are publishing the post, I think it would be better to have posts with titles.